ntp-proto/src/peer.rs

```diff
@@ -227,13 +228,13 @@ impl PeerSnapshot {
         // Note, this can only ever be an issue if the peer is not using
         // hardware as its source, so ignore reference_id if stratum is 1.
         if self.stratum != 1 && self.reference_id == self.our_id {
-            debug!("Peer rejected because of detected synchornization loop");
+            warn!("Peer rejected because of detected synchornization loop");
```

Comment:
This can happen during normal operation and is not necessarily the result of any misconfiguration, so we should probably log this as just info.

Reply:
choice:info